client/ayon_substancepainter/plugins/publish/collect_staging_dir_for_textures.py

```diff
@@ -0,0 +1,39 @@
+import copy
+
+import pyblish.api
+from ayon_core.pipeline import publish
+
+
+class CollectStagingDirExportConfig(pyblish.api.InstancePlugin):
+    """Collect Staging Dir for Export Config to export the textures"""
+
+
+    label = "Collect Staging dir for Export Config"
+    hosts = ["substancepainter"]
+    families = ["textureSet", "textures", "image"]
+    order = pyblish.api.CollectorOrder + 0.491
+
+    def process(self, instance):
+        export_config = copy.deepcopy(instance.data["exportConfig"])
+        export_config["exportPath"] = publish.get_instance_staging_dir(instance)
+        instance.data["exportConfig"] = export_config
+        instance.data.update(instance.data["exportConfig"])
```

Comment:
This is wrong. This should just be:
```suggestion
        instance.data["exportConfig"]["exportPath"] = instance.data["stagingDir"]
```
In this case we do **want** to update it in-place.

Reply:
![image](https://github.com/user-attachments/assets/b13dde78-4036-4ca1-a29c-12ec625f178f)

Reply:
It should run **after** Collect Managed Staging Dir.